### PR TITLE
[DNM] remove repository definition for Apache

### PR DESCRIPTION
**Pull request by ktdreyer**
It's better to rely on the distro Apache packages. See thread on ceph-devel, "Ceph's custom apache: ok to drop?" http://comments.gmane.org/gmane.comp.file-systems.ceph.devel/22368

### CODE REVIEW

**Comment by andrewschoen** [comment]
@ktdreyer I'll probably take over this PR as I've picked up tracker issue #10551.

@SandonV would you mind looking over my comments here as a sanity check?  I'm not familiar with chef much at all.

Also, sorry to comment on the commit and not the diff.  Github makes that a bit confusing sometimes...

**Comment by SandonV** [comment]
@andrewschoen your comments look good. Just a couple extra things:

We need to use the action : remove on httpd (or if we are absolutely 100% sure that the default apache2 is newer then do upgrade instead of just a regular install) to insure that we are removing the existing packages installed on machines (see qemu).

Also no new packages are being brought in? that doesn't seem quite right unless we are using something built into apache.

**Comment by andrewschoen** [comment]
@SandonV I'm not 100% sure.  So, I'll :remove and then install.

AFAIK, no new packages are needed for apache.  We will be moving mod_fastcgi to mod_proxy_fcgi though.

**Comment by ktdreyer** [comment]
The details regarding the mod_fastcgi -> mod_proxy_fcgi switch are at http://tracker.ceph.com/issues/10551 . There is also a ticket for doing mod_proxy_fcgi builds, here: http://tracker.ceph.com/issues/10808

**Comment by ktdreyer** [comment]
To clarify: there are two separate issues that need to be fixed for Teuthology's RGW testing.

1. RGW uses a forked copy of Apache, and it should stop using Ceph's fork and start using the distro packages instead. This is what is discussed and confirmed in this mailing list thread: http://comments.gmane.org/gmane.comp.file-systems.ceph.devel/22368
2. RGW uses mod_fastcgi, and it should use mod_proxy_fcgi instead. This is what is tracked in http://tracker.ceph.com/issues/10551 "test RGW with mod_proxy_fcgi instead of mod_fastcgi" and http://tracker.ceph.com/issues/10808 "Must build packages for mod_proxy_fcgi for all distros"

In other words, for the mod_proxy_fcgi switch, we're going to need packages, but for the Apache switch, there's nothing blocking that, and we can implement that today.

**Comment by ktdreyer** [comment]
PR #8 has been merged, so I'm closing this. Thanks @andrewschoen
